daml-lf/interface/src/main/scala/com/digitalasset/daml/lf/iface/reader/InterfaceReader.scala

```diff
@@ -195,7 +195,7 @@ object InterfaceReader {
     : InterfaceReaderError \/ ImmArraySeq[(Ref.Name, Type)] =
     fields.toSeq traverseU { case (fieldName, typ) => type_(ctx, typ).map(x => fieldName -> x) }
 
-  private def type_(
+  def type_(
```

Comment:
@S11001001, @cocreature: what would be a bit more descriptive name for this?

Reply:
How about `toIfaceType`?

Reply:
Renamed to `toIfaceType`. Happy to change it to something else, I don’t care much for the name here.